resources/js/views/RootContainer.vue

```diff
@@ -1,23 +1,30 @@
 <template>
     <div class="root-container">
         <Nav />
-
+        <div v-show="showEmailConfirmedBanner"><WelcomeBanner /></div> 
```

Comment:
No need to wrap this in a div. You can use v-show on the component or else put v-show inside it